### PR TITLE
[JENKINS-45915] - Fix a ConsoleNote Base64 decoding bottleneck by switching to Java 8 Base64 Impl

### PR DESCRIPTION
**Pull request by svanoort**
See [JENKINS-45915](https://issues.jenkins-ci.org/browse/JENKINS-45915).

The decoder implementation is a drop-in replacement of a spec, and should be covered by existing tests where needed.

This is similar to #2855 (which is replacing Trilead implementations)

### Proposed changelog entries

* Improve Performance When Reading the ConsoleText from a Run

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests

### CODE REVIEW

**Review by johnou — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
@reviewbybees

**Review by oleg-nenashev — APPROVED** [review]
🐝

**Review by daniel-beck — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
@reviewbybees done

File: core/src/main/java/hudson/util/UnbufferedBase64InputStream.java [hunk 1]
```diff
@@ -22,6 +21,7 @@
     private byte[] decoded;
     private int pos;
     private final DataInputStream din;
+    private static final Base64.Decoder decoder = Base64.getMimeDecoder();
```
Comment (jglick) [inline core/src/main/java/hudson/util/UnbufferedBase64InputStream.java]: This seems to be thread-safe.

**Review by jglick — APPROVED** [review]
There are a ton of other usages of Commons Codec and Trilead versions of this utility which should also be replaced, but this is a good start I guess.

File: core/src/main/java/hudson/util/UnbufferedBase64InputStream.java (hunk 1 above)
Comment (johnou) [inline core/src/main/java/hudson/util/UnbufferedBase64InputStream.java]: Yep, it is mentioned in the java doc.

**Pull request merged**